pgcontents/tests/test_pgcontents_api.py

```diff
@@ -163,6 +163,45 @@ def test_list_checkpoints_sorting(self):
             )
         )
 
+    # ContentsManager has different behaviour in notebook 5.5+
+    # https://github.com/jupyter/notebook/pull/3108...it now allows
+    # non-empty directories to be deleted.
+    #
+    # PostgresContentsManager should continue to work the old way and
+    # prevent non-empty directories from being deleted, since it doesn't
+    # support backing up the deleted directory in the OS trash can.
+    # FileContentsManager should allow non-empty directories to be deleted.
+    def test_delete_non_empty_dir(self):
+        if issubclass(self.notebook.contents_manager_class,
```

Comment:
I'd probably use `isinstance(self.notebook.contents_manager, ...)` for these checks rather than `issubclass`.